packages/timeline-state-resolver/src/integrations/pharos/$schemas/actions.json

```diff
@@ -0,0 +1,10 @@
+{
+	"$schema": "../../../$schemas/action-schema.json",
+	"actions": [
+		{
+			"id": "clearQueue",
```

Comment:
Something else to question for cleanup, why do we have to explicitly do this clear? Shouldnt that be handled for us by the conductor due to the changing timeline?

Reply:
I agree, I don't see why this was in makeReady in the first place. I didn't really look at what the individual makeReady functions did but I'm just going to remove this one.

Reply:
Agreed, this one could probably be removed 👍